vms/platformvm/txs/builder/camino_helpers_test.go

```diff
@@ -172,15 +172,20 @@ func newCaminoEnvironment(postBanff bool, caminoGenesisConf api.Camino) *caminoE
 	return env
 }
 
-func newCaminoBuilder(postBanff bool, state state.State) (*caminoBuilder, *versiondb.Database) {
+func newCaminoBuilder(t *testing.T, postBanff bool, state state.State) (*caminoBuilder, *versiondb.Database) {
 	var isBootstrapped utils.Atomic[bool]
 	isBootstrapped.Set(true)
 
 	config := defaultCaminoConfig(postBanff)
 	clk := defaultClock(postBanff)
 
 	baseDB := versiondb.New(memdb.New())
-	ctx, _ := defaultCtx(baseDB)
+	ctx := snowtest.Context(t, snowtest.PChainID)
+	m := atomic.NewMemory(baseDB)
+	msm := &mutableSharedMemory{
+		SharedMemory: m.NewSharedMemory(ctx.ChainID),
+	}
+	ctx.SharedMemory = msm
```

Comment:
wouldn't be more minimal and less repetitive if we just changed the inner part of `defaultCtx` to just include this part.
```go 
ctx := defaultCtx(baseDB)
...
func defaultCtx(db database.Database) (*snow.Context){
	ctx := snowtest.Context(t, snowtest.PChainID)
	m := atomic.NewMemory(baseDB)
	msm := &mutableSharedMemory{
		SharedMemory: m.NewSharedMemory(ctx.ChainID),
	}
	ctx.SharedMemory = msm
    return ctx
}
```